pytest_twisted.py

```diff
@@ -326,3 +327,17 @@ def pytest_configure(config):
     )(blockon)
 
     reactor_installers[config.getoption("reactor")]()
+
+
+def use_asyncio_selector_if_required(config):
+    # https://twistedmatrix.com/trac/ticket/9766
+
+    if (
+        config.getoption("reactor", "default") == "asyncio"
+        and sys.platform == 'win32'
+        and sys.version_info >= (3, 8)
+    ):
+        import asyncio
+
+        selector_policy = asyncio.WindowsSelectorEventLoopPolicy()
+        asyncio.set_event_loop_policy(selector_policy)
```

Comment:
I'm writing up some docs on the topic too (and fixed the missing trailing newline) but it feels like there's some thing to provide users with that isn't copy/paste.  But, this isn't a `pytest-twisted` issue...  and it could be run regardless of version.  I can't decide how I feel about this.